app/controllers/admin_applications_controller.rb

```diff
@@ -0,0 +1,39 @@
+class AdminApplicationsController < ApplicationController
+  before_action :require_admin
+  before_action :get_application
+  before_action :skip_validation, except: [:destroy]
+
+  def approve
+    @application.update_attributes(status: "approved")
```

Comment:
`update_attributes` can be shortened to just `update` unless your using an old Rails version. Do you use rubocop? It usually suggests things like this.
I'd also use `update!` here, because you don't handle the case of a failed application. If for some reason the update fails, I'd want this to explode right away and cause and exception I'll get notified of instead of sending wrong information to the user.

Reply:
@A5308Y thanks a lot for the review! I do have Rubocop and I must be honest and say that I am ignoring it but it's not suggesting me this change. Anyway I add your suggestions.